libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3656,7 +3656,7 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
         magnitude: 18,
       },
     ],
-    explorerViews: [blockscoutExplorerView("https://crypto.org/explorer")],
+    explorerViews: [blockscoutExplorerView("https://cronos.org/explorer/")],
```

Comment:
Do you want to keep the last `/` here ? It's not done like this for the other ones
```suggestion
    explorerViews: [blockscoutExplorerView("https://cronos.org/explorer")],
```

Reply:
Good catch! Ty 👍🏾